test.json

```diff
@@ -0,0 +1,240 @@
+{
+    "spdxVersion": "SPDX-2.1",
+    "documentNamespace": "https://spdx.org/spdxdocs/spdx-example-444504E0-4F89-41D3-9A0C-0305E82C3301",
+    "creationInfo": {
+        "creators": [
```

Comment:
This file is probably a leftover from development, I think it can be deleted?!

Reply:
Oh yes